src/components/tag/tag.mod.css

```diff
@@ -3,9 +3,9 @@
 .contentWrapper {
   display: flex;
   align-items: center;
-  height: var(--token-size-height-tag);
+  min-height: var(--token-size-height-tag);
   border-radius: var(--token-border-radius-tag);
-  padding: 0 var(--spacing-8);
+  padding: 5px var(--spacing-8);
```

Comment:
Yeah. By removing height: 26, you end up with a computed height of 14px (comes from line-height). So then, to make up for this diff, we need to add 10 px of padding, (since 2 px comes from the border), resulting in a total height again of 26 🤦‍♂️